### ISSUE TITLE
How to run certain code at specific frame of animation?

### ISSUE THREAD

**Issue by Kardane** [Provim-Gaming/nylon#2]
I'm making monster with its own attack animation.
I found that I can only run a code at end of animation, not at in middle of animation.

> Animation frame effects: Certain effects that can be conditionally applied on a specific frame of an animation. For example, running a command, playing a sound or changing the variant.
How can I run codes in middle of animation as written at readme?

**Comment by tomalbrc** [comment]
Nylon supports the Command keyframes in AnimatedJava (https://animated-java.dev/docs/animations under "Commands")

**Comment by Kardane** [comment]
O_o So Nylon model also run commands as well? I didnt know that

**Comment by Wesley1808** [comment]
Yeah, animation (key) frame effects are referring to the animated-java feature thats part of the model file.
From Nylon 0.9.13 and forward you will also be able to listen for animation frame updates in code.

An example of this:
```java
this.holder.getAnimator().playAnimation("melee", 10, (frame) -> {
    switch (frame) {
        case 16 -> System.out.println("Frame 16");
        case 24 -> System.out.println("Frame 24");
    }
});
```

**Comment by Kardane** [comment]
Nice!
